src/libcommon/utility/utility.cpp

```diff
@@ -640,6 +640,11 @@ QString CommonUtility::getRelativePathFromHome(const QString &dirPath) {
     return QDir::toNativeSeparators(name);
 }
 
+bool CommonUtility::isFileSizeMismatchDetectionEnabled() {
```

Comment:
```suggestion
constexpr bool CommonUtility::isFileSizeMismatchDetectionEnabled() {
```

Reply:
I don't think this is possible as the return value cannot be determine at compilation time?

Reply:
Oh, yes, sorry ! My mistake.